detections/endpoint/windows_expand_cabinet_file_extraction.yml

```
@@ -0,0 +1,85 @@
+name: Windows Expand Cabinet File Extraction
+id: 4e3e3b8c-6d3a-4b47-9f5a-9e3e0a0a6f2f
+version: 1
+date: '2025-09-18'
+author: Michael Haag, Splunk
+status: production
+type: TTP
+description: |
+  Detects usage of expand.exe to extract Microsoft Cabinet (CAB) archives, with
+  emphasis on extractions into `C:\\ProgramData` or similar staging locations. In
+  recent APT37 activity, a CAB payload (e.g., wonder.cab) was expanded into
+  ProgramData prior to persistence and execution. This behavior is a strong signal
+  for ingress tool transfer and staging of payloads.
+data_source:
+- Sysmon EventID 1
+- Windows Event Log Security 4688
+- CrowdStrike ProcessRollup2
+search: |
+  | tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime
+  from datamodel=Endpoint.Processes
+  where Processes.process_name="expand.exe"
+    (Processes.process="* -F:* *" OR Processes.process="* /F:* *")
+    (Processes.process="*.cab*" OR Processes.process="*\\ProgramData\\*")
```

Expansion can be legitimate for so just having a `.cab` can and will lead to FP.
Also the `.cab` is not required, as long as its a legit cab it will be extracted.

Hence I suggest you look for suspicious paths (maybe add more than `ProgramData`). If you want to look for any expansion then create a separate anomaly or hunting rule.

```suggestion
    Processes.process="*\\ProgramData\\*"
```